ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/structured_configs/bgp-peer-groups-structured-config-1.yml

```diff
@@ -168,15 +168,16 @@ prefix_lists:
         action: permit 192.168.255.0/24 eq 32
       20:
         action: permit 192.168.254.0/24 eq 32
+ip_extcommunity_lists:
```

Comment:
we are not converting to lists yet. This would fail if someone used custom_structured_configuration

Reply:
Addressed in 6110996